### ISSUE TITLE
Android Tmdb Login

### ISSUE THREAD

**Issue by 4face-studi0** [4face-studi0/CineScout#50]
Blocked by #52 and #53

- [x] Define `DrawerViewModel` that provides profile info if any or none
- [x] the ViewModel must be able to start the login / logout
- [ ] the ViewModel can now hold screen states **( this is not needed now )**
